packages/react/src/components/Suggestion/Suggestion.stories.tsx

```diff
@@ -52,6 +52,7 @@ async function testSuggestion(el: HTMLElement) {
 }
 
 const DATA_PLACES = [
+  'Lillestrøm',
```

Comment:
😶‍🌫️ Nothing to see here...

Reply:
![gotcha-wink](https://github.com/user-attachments/assets/b1e247d9-c65c-443b-b4f3-9f70fac4b0d5)